draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,28 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers {#subscription-id}
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a track, it associates a Request ID with the request.  The
+corresponding Subscription Request ID is equal to `Request ID << 1`.
```

Comment:
there is no Susbcription Request ID. So may be we should remove the word "Subscription"

Reply:
I'm defining Subscription Request ID here.  It can be used anywhere to refer to this subscription (control messages and data streams).

Reply:
> It works symmetrically for publish, because Publish also includes a Request ID.

Interesting.  So the publisher effectively just has two IDs it knows at the moment it sends a PUBLISH message, and it can pick either?  I am not sure I see the justification for having two separate IDs.

Reply:
>  So the publisher effectively just has two IDs it knows at the moment it sends a PUBLISH message, and it can pick either? 

There _may_ be only one ID -- PSID is not required for publish.  You can publish using Subscription Request ID too, it just has the same control and data reordering problems that PSID does.

The justification for using Request ID in Publish is so a receiver can limit the number of publishes using MAX_REQUEST_ID.

Reply:
I renamed it `Request Subscription ID`, and tried to make it clear that this is a new term, and that Subscription ID = RSID or PSID